mftf/2.3/commands/mftf.md

```diff
@@ -90,6 +90,12 @@ vendor/bin/mftf build:project [--upgrade]
 | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `-u`, `--upgrade` | Upgrades existing MFTF tests according to requirements of the last major release. Specifying this flag upgrades only those tests in the default location. Example: `build:project --upgrade`. |
 
+You can use options to set up particular configuration parameters, since the `build:project` command runs [`setup:env`][setup] as well.
```

Comment:
Will change to:
You can include options to set configuration parameter values for your environment since the project build process also [sets up the environment][setup].